selenium_test.py

```diff
@@ -0,0 +1,222 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+import pandas as pd
+import time
+
+KEYWORDS_FILE = 'textfile.txt'
+RANKED_OUTPUT_FILE = 'Output.csv'
+START_YEAR = "1917"
+END_YEAR = "1971"
+
+
+class resultLists:
+    title_list = []
+    description_list = []
+    category_list = []
+    keyword_list = []
+    link_list = []
+
+lists = resultLists()
+
+
+
+def read_keywords():
+    with open(KEYWORDS_FILE) as fd:
+        return [keyword.strip() for keyword in fd.readlines()]
+
+
+def economist_scraper(keyword, startyear, endyear):
```

Comment:
In Python, you can have [default arguments](https://beginnersbook.com/2019/06/python-function-arguments-default-keyword-and-arbitrary/), which means you can write:

`def economist_scraper(keyword, startyear-START_YEAR, endyear=END_YEAR):`

Also, a better name for this function is `scrape_economist`. Get in the habit of naming functions with verbs that describe what they do. The exception to this is if they calculate something, in which case the function should be named after what it calculates (eg. `squareroot`).